environment.yml

```diff
@@ -3,4 +3,6 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - nextflow=20.01
+  - nextflow=20.10
+  - samtools
+  - conda<4.7
```

Comment:
Any reason for teh NF increase and especially adding the other two software packages?

samtools should be handled by the environmen files I guess. Conda I don't know. do you have issues with it?